srcpkgs/python3-cheetah3/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'python3-cheetah3'
+pkgname=python3-cheetah3
+version=3.2.0
+revision=1
+wrksrc="Cheetah3-${version}"
+build_style=python3-module
+pycompile_module="Cheetah"
+hostmakedepends="python3-setuptools"
+makedepends="python3-devel"
+short_desc="Python-powered template engine and code generator"
+maintainer="Piotr Wójcik <chocimier@tlen.org>"
+license="MIT"
+homepage="http://www.cheetahtemplate.org/"
+distfiles="${PYPI_SITE}/C/Cheetah3/Cheetah3-${version}.tar.gz"
+checksum=0c8d1a2494eaad548544cb2c317514d5279db0ed569e34516b787d5e12b41589
+alternatives="python-cheetah:/usr/bin/cheetah:/usr/bin/cheetah3
```

Comment:
Second field can be just the basename if they dirname is the same as the last field

see busybox

Reply:
changed

Reply:
ah, wrong

Reply:
fixed